<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPanel</class>
 <widget class="QMainWindow" name="TestPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Laser Comms</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="laserCommState">
      <item>
       <property name="text">
        <string>Excellent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Good</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Offline</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Camera Comms</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cameraCommsState">
      <item>
       <property name="text">
        <string>Excellent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Good</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Offline</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Laser Op State</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="laserOpState">
      <item>
       <property name="text">
        <string>Online</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Offline</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Camera Op State</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cameraOpState">
      <item>
       <property name="text">
        <string>Online</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Offline</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Rover Op State</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="roverOpState">
      <item>
       <property name="text">
        <string>Acquired</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Not Acquired</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Laser Active</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="laserActiveState">
      <item>
       <property name="text">
        <string>Active</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Not Active</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
